### PR TITLE
Add EntityItemPickupEvent and EntityTravelToDimensionEvent

### PR DESCRIPTION
**Pull request by williambl**
Both needed for Forestry (#94)

### CODE REVIEW

**Comment by williambl** [comment]
Okay, this is ready now.

File: patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java [hunk 1]
```diff
@@ -217,6 +219,19 @@ public static boolean onProjectileImpact(ThrownEntity throwable, HitResult ray)
 		return MinecraftForge.EVENT_BUS.post(new ProjectileImpactEvent.Throwable(throwable, ray));
 	}
 
+	public static boolean onTravelToDimension(Entity entity, DimensionType dimensionType) {
+		EntityTravelToDimensionEvent event = new EntityTravelToDimensionEvent(entity, dimensionType);
+		return !MinecraftForge.EVENT_BUS.post(event);
+		/* Forge has this thing:
```
Comment (TheGlitch76) [inline patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java:225]: why comment this out?

Comment (williambl) [reply patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java:225]: `StorageMinecart` doesn't seem to have `dropContentsWhenDead` - i assume it's a forge method

Comment (williambl) [reply patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java]: It's the same behaviour as forge now - what do you mean by adding a // TODO? What should it say?

**Pull request merged**